repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ProjectConfiguration.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.pipeline.datastorage.dto
+
+import java.nio.file.{ Path, Paths }
+
+import org.mongodb.scala.bson.collection.immutable.Document
+import play.api.libs.json._
+
+case class ProjectFileConfiguration(path: Path, inputs: List[FileParameter])
+
+object ProjectFileConfiguration {
+  implicit object ProjectFileConfigurationFormat extends Format[ProjectFileConfiguration] {
+    override def reads(json: JsValue): JsResult[ProjectFileConfiguration] = JsSuccess(
+      ProjectFileConfiguration(Paths.get((json \ "path").as[String]), (json \ "inputs").as[List[FileParameter]])
```

Comment:
You must write a formatter for `Path`. You should not convert it to string / from string everywhere manually.